src/Akeneo/Pim/Structure/Bundle/Query/InternalApi/Attribute/GetBlacklistedAttributeJobExecutionId.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Structure\Bundle\Query\InternalApi\Attribute;
+
+use Doctrine\DBAL\Connection;
+
+final class GetBlacklistedAttributeJobExecutionId
+{
+    private Connection $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function forAttributeCode(string $attributeCode): ?int
+    {
+        $sql = <<<SQL
+SELECT ab.cleanup_job_execution_id FROM `pim_catalog_attribute_blacklist` ab
+WHERE ab.attribute_code = :attribute_code;
+SQL;
```

Comment:
```suggestion
SELECT cleanup_job_execution_id 
FROM `pim_catalog_attribute_blacklist`
WHERE attribute_code = :attribute_code;
SQL;
```

We only have on table here I don't know if it's relevant to have alias